build/source/driver/multi_driver.f90

```diff
@@ -219,6 +219,7 @@ program multi_driver
 type(hru_d),allocatable          :: upArea(:)                  ! area upslope of each HRU 
 ! general local variables        
 integer(i4b)                     :: ivar                       ! index of model variable
+integer(i4b)                     :: nSoil                      ! Maximum Number of Soil Layers
```

Comment:
suggest use
`integer(i4b),parameter    :: maxSoilLayers=10  ! maximum number of soil layers

Reply:
This has been used.